.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
So this .vscode folder is not needed to be added to the repo. These are your private settings which are not directly related to the project, even though they are in the same folder.

You need to exclude this folder by creating a .gitignore file and just add .vscode to it.

For an example, see the .gitingore file in the lessons repository :)